test/node-rpc-test.js

```diff
@@ -68,11 +69,29 @@ describe('RPC', function() {
     assert.deepEqual(info.localservicenames, ['NETWORK', 'WITNESS']);
   });
 
+  it('should rpc getpeerinfo', async () => {
+    const info = await nclient.execute('getpeerinfo', []);
+    assert.deepEqual(info, []);
+  });
+
+  it('should rpc getconnectioncount', async () => {
+    const connectionsCnt = await nclient.execute('getconnectioncount', []);
+    assert.strictEqual(connectionsCnt, 0);
+  });
+
+  it('should rpc getchaintips', async () => {
+    const info = await nclient.execute('getchaintips', []);
+    assert.strictEqual(util.revHex(node.chain.tip.hash), info[0].hash);
+  });
```

Comment:
Same for this test, maybe we should start another `describe` subsection for blockchain tests? I dunno, your choice (we can discuss offline, maybe how to make these test files look cleaner, etc)

Anyway, this test so far is missing an `assert.strictEqual(info.length, 1)` -- otherwise why are you only checking the hash for `info[0]` ?  Don't we want to make sure `getchaintips` only returns the one chain we know about so far?

And then, yes we should add another chain tip and test again. How I think we can do this is by mining a block that competes with the current tip:

```js
const entry = await node.chain.getEntry(chain.tip.height - 1);
const block = await node.miner.mineBlock(entry);
await node.chain.add(block);
```

After this I believe you will now have a second chain tip whose hash is `block.hash()`